backend/services/automations/src/executions/checkIsWaitingActionTarget.ts

```
@@ -0,0 +1,85 @@
+import { IModels } from '@/connectionResolver';
+import { IAutomationWaitingActionDocument } from '@/mongo/waitingActionsToExecute';
+
+function accessNestedObject(obj: any, keys: string[]) {
+  return keys.reduce((acc, key) => acc && acc[key], obj) || '';
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)